<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>IMContactDetails</class>
 <widget class="QWidget" name="IMContactDetails" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>24</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="imContactIdLabel" >
       <property name="minimumSize" >
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text" >
        <string>Contact ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="removeButton" >
       <property name="maximumSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="text" >
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="protocolPixmap" >
       <property name="pixmap" >
        <pixmap resource="../wengophone.qrc" >:/pics/protocols/aim.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../wengophone.qrc" />
 </resources>
 <connections/>
</ui>
